generators/service-appid/templates/swift/instrumentation.swift

```diff
@@ -1,15 +1,15 @@
 import LoggerAPI
+import CloudEnvironment
 import BluemixAppID
-import Credentials
+import class Credentials.Credentials
 
 // For more details on AppID usage, see
 // https://github.com/ibm-cloud-security/appid-serversdk-swift#protecting-web-applications-using-webappkituracredentialsplugin
 // Use kituraCredentials to protect your endpoints/routes
 
-var webappKituraCredentialsPlugin: WebAppKituraCredentialsPlugin!
-var kituraCredentials: Credentials!
-
-func initializeServiceAppid() throws {
+func initializeServiceAppid(cloudEnv: CloudEnv) throws -> Credentials {
+    let webappKituraCredentialsPlugin: WebAppKituraCredentialsPlugin
+    let kituraCredentials: Credentials
```

Comment:
If you move these declarations below the guard you won't need the type annotations since they would be inferred by the assignments.

Reply:
done